### PR TITLE
[FIX] remove static self::plugin() calls

### PR DESCRIPTION
**Pull request by chfsx**

### CODE REVIEW

File: vendor/composer/autoload_real.php [hunk 1]
```diff
@@ -22,6 +22,8 @@ public static function getLoader()
             return self::$loader;
         }
 
+        require __DIR__ . '/platform_check.php';
```
Comment (ferishili) [inline vendor/composer/autoload_real.php:25]: I saw that you have removed this platform checker before, therefore I would assume that adding it here is part of the plan? Right?

File: vendor/composer/platform_check.php
Comment (ferishili) [inline vendor/composer/platform_check.php:1]: The same assumption also applies for this one!

File: vendor/composer/autoload_real.php (hunk 1 above)
Comment (chfsx) [reply vendor/composer/autoload_real.php:25]: this is auto-generatoed by composer, unfortunately it seems to depend on which version of composer the classmap is generated with.

File: vendor/composer/platform_check.php
Comment (chfsx) [reply vendor/composer/platform_check.php:1]: same here

**Pull request merged**